### DISCUSSION TITLE
Emulation usage

### DISCUSSION THREAD

**Discussion by JohnLukeFaris** [Key2Joy/Key2Joy#85] [Q&A]
I am using the xemu emulator to play Halo Combat Evolved, but the problem is that the keys used to look around are i-j-k-l. The game is insufferable, but I'm not a huge fan of console, so I'm trying to figure out how to get it working with my keyboard. I was wondering if anyone would be able to tell me if this is the right place to be looking, or if I'm completely wrong. I've also seen something about hooking the emulator up to Steam, but I haven't tried that yet. Thank you so much to anyone who has anything to say!

**Comment by luttje** [comment]
Hey there! Maybe this tool can help. Try this version: https://github.com/luttje/Key2Joy/releases/tag/0.6.0

Note that there is some annoying bugs that I have yet to resolve, so expect a quite jank experience.

After extracting the tool somewhere, open it and create a mapping from your mouse movement to a simulated GamePad Stick action:
1. Create a new mapping
2. Set the trigger to 'Mouse Move Event'
3. Select the Direction 'Any'
4. Set the action to 'GamePad Stick Simulation'
5. Set Stick to 'Right'
6. Leave the rest of the settings as is
7. Try if this works in the game. 

If you can look around but not move, then you may also need to create mappings for W, A, S and D to bind to the Left GamePad Stick directions. Same for all other buttons you need to use.

Also if it is within your budget, I think the more cost effective way (by saving time) would be to just buy the game for PC: https://store.steampowered.com/app/976730/Halo_The_Master_Chief_Collection/

Hope this helps!
